src/anyon/anyon.jl

```diff
@@ -660,13 +776,14 @@ end
 """
     run_job(qpu::AnyonYukonQPU, circuit::QuantumCircuit, shot_count::Integer)
 
-Run a circuit computation on a `QPU` service, repeatedly for the specified
-number of repetitions (shot_count).
-Returns the histogram of the completed circuit calculations, along with the 
-simulation's execution time (in milliseconds) or an error
-message.
-If the circuit received in invalid - for instance, it is missing a `Readout` -
-it is not sent to the host, and an error is throw.
+Submit a circuit to a `QPU` service for execution. The number of circuit executions is
+specified by `shot_count`.
+
+Returns a histogram of the circuit measurement outcomes along with the 
+simulation's execution time (in milliseconds) or an error message.
+
+If `circuit` is invalid - for instance, if it is missing a `Readout` - it is not sent to the
```

Comment:
There are other ways it can be invalid if a user purposefully avoids the normal compilation stage. Let's be specific about what checks are always performed

Reply:
perhaps we should always check for native gates as well, not for this MR